dscreator/sources/base.py

```diff
@@ -37,6 +37,12 @@ def __post_init__(self):
 class FeatureBase:
     array_list: List[NamedArray]
 
+    def __add__(self, na_list):
+        for na1 in self.array_list:
+            for na2 in na_list:
+                if na1.variable_name == na2.variable_name:
+                    na1.values = na1.values + na2.values
+        return self.array_list
```

Comment:
I would just remove this function this can be handled in Namedtrajectory. Also, what about variables in `na_list` that is not in  self.array_list.